habitat-lab/habitat/tasks/rearrange/rearrange_sim.py

```diff
@@ -48,18 +48,16 @@ class RearrangeSim(HabitatSim):
     def __init__(self, config: Config):
         if len(config.agents) > 1:
             with read_write(config):
-                all_new_sensor_names = []
                 for agent in config.agents:
                     agent_cfg = config[agent]
-                    for orig_sensor_name in config.agent_0.sensors:
-                        full_name = f"{agent}_{orig_sensor_name}"
-                        orig_sensor_id = config[orig_sensor_name].uuid
-                        new_sensor_cfg = config[orig_sensor_name].clone()
-                        new_sensor_cfg.uuid = f"{agent}_{orig_sensor_id}"
-                        config[full_name] = new_sensor_cfg
-                        all_new_sensor_names.append(full_name)
-
-                config.agent_0.sensors = all_new_sensor_names
+                    sensor_keys = list(agent_cfg.sim_sensors.keys())
```

Comment:
```suggestion
                    sensor_keys = agent_cfg.sim_sensors.keys()
```
No reason this needs to be a list. It is only iterated over once.